homeassistant/components/shelly/strings.json

```diff
@@ -8,7 +8,7 @@
           "host": "[%key:common::config_flow::data::host%]"
         },
         "data_description": {
-          "host": "The hostname or IP address of the Shelly device to connect to."
+          "host": "The hostname or IP address of the Shelly device to connect to.\nYou can also specify a port with 'ip:port' syntax."
```

Comment:
I think this will look better in one line:

```suggestion
          "host": "The hostname or IP address of the Shelly device to connect to. You can also specify a port with 'IP:PORT' syntax."
```

![obraz](https://github.com/home-assistant/core/assets/478555/e7b00101-5bc3-4844-81ef-4ecf33a0b1f1)